crates/rust-analyzer/tests/rust-analyzer/main.rs

```diff
@@ -527,7 +527,7 @@ version = \"0.0.0\"
 #[test]
 fn out_dirs_check() {
     if skip_slow_tests() {
-        return;
+        // return;
```

Comment:
Looks like you left this in by accident

Reply:
Good catch, fixed https://github.com/rust-analyzer/rust-analyzer/pull/8584.

I wonder if there's a better way to convenently run a single slow test from the editor that uncommenting the thing...

Reply:
Maybe an env var that RA can set similar to what we do for updating expect tests via command?

Reply:
The way I did :

```rust
fn out_dirs_check() {
    if skip_slow_tests() {
        // TODO:
        // return;
    }
}
```

Reply:
Cool, didn't thought about that!